src/App.js

```diff
@@ -53,6 +96,10 @@ function App() {
             <div className={styles.container}>
               <h1 className={styles.h1}>Todo list</h1>
               <AddTodoForm onAddTodo={addTodo} />
+              <button type="button" onClick={() => setOrder(!order)}>
+                {" "}
+                button
```

Comment:
Maybe call this button 'Sort'. And try to add a way to bring it back its original state.
Also, think about what happens to the sorted list when add a new `todoItem`. The list should probably remain in the sorted order A-Z or Z-A if you add a new item so you'll have to re-sort after each item is added.

Reply:
Yes I noticed after I submitted the assignment that I had accidentally left the button like that 🤦🏽‍♀️